docker-compose.yml

```diff
@@ -45,16 +46,16 @@ services:
       interval: 5m
       start_period: 1m
 
+  orchestrator:
+    <<: *app_base
+    command: bundle exec sidekiq -r ./extras/dynflow-sidekiq.rb -c 1 -q dynflow_orchestrator
+    hostname: orchestrator.example.com
+    ports: []
+
   worker:
     <<: *app_base
-    command: bundle exec rake dynflow:executor
-    hostname: worker.example.com
-    ports:
-      - 9999
-    healthcheck:
-      test: "ps ax | grep -v grep | grep dynflow_executor"
-      interval: 1m
-      start_period: 1m
+    command: bundle exec sidekiq -r ./extras/dynflow-sidekiq.rb -c 15 -q default,rex
```

Comment:
Isn't the rex queue full named `remote_execution` instead of `rex`?

Reply:
It is, thank you for spotting that